src/components/MediaDevices/MicCheck.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import {
+  useDevices,
+  useLocalSessionId,
+  useAudioTrack,
+} from '@daily-co/daily-react';
+import { Link } from 'react-router-dom';
+
+import { AudioAnalyser } from '../shared/AudioVisualiser/AudioAnalyser.ts';
+import { AudioVisualiser } from '../shared/AudioVisualiser/AudioVisualiser.tsx';
+
+import { Card } from '../shared/Card/Card';
+import { TroubleShooting } from '../shared/TroubleShooting/TroubleShooting';
+import { logger } from '../../utils/Logger';
+
+export const MicCheck: React.FC = () => {
+  const localSessionId = useLocalSessionId();
+  const audioTrack = useAudioTrack(localSessionId ? localSessionId : '');
```

Comment:
The ternary should be unnecessary ☺️ 
```suggestion
  const audioTrack = useAudioTrack(localSessionId);
```